src/app/main/component/shared/components/events-list-item/events-list-item.component.ts

```diff
@@ -238,6 +254,34 @@ export class EventsListItemComponent implements OnInit, OnDestroy {
 
   public subscribeToLangChange(): void {
     this.langChangeSub = this.localStorageService.languageSubject.subscribe(this.bindLang.bind(this));
+    this.localStorageService.languageBehaviourSubject.subscribe((lang: string) => {
+      this.currentLang = lang;
+      this.datePipe = new DatePipe(this.currentLang);
+      this.newDate = this.datePipe.transform(new Date(), 'MMM dd, yyyy');
+    });
+  }
+
+  cutTitle(): string {
+    if (this.event.title.length > 40) {
+      return this.event.title.slice(0, 30) + '...';
+    } else {
+      return this.event.title;
+    }
```

Comment:
use ternary operator here
return this.event.title.length > 40 ? this.event.title.slice(0, 30) + '...'  :  this.event.title